app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt

```diff
@@ -44,6 +45,21 @@ class NewCreatePinFragment :
         }
     }
 
+    private fun UiKitToolbar.initToolbar() {
+        setNavigationOnClickListener { popBackStack() }
+        setOnMenuItemClickListener {
+            when (it.itemId) {
+                R.id.helpItem -> {
+                    // TODO PWN-4362 open help
```

Comment:
Why did you move this to another ticket? 
Is there any issue to open feedback and support here?

Reply:
Just changed the message due to @kabramovp2p comment. Ticket is the same

Reply:
@kuznetsov why you are leaving TODO here? 
Why you are not opening "Support and feedback" screen as it said in the ticket?

Reply:
@kuznetsov you can put implementation here, I asked Nina about it yesterday.
Open Intercom window on help item click

Reply:
Thanks guys, opened Intercom from here